src/components/LanguageOption.tsx

```diff
@@ -146,6 +146,8 @@ export default function LanguageOption(props: LanguageOptionProps) {
       .then(() => {
         const metadataExisted = Boolean(metadata?._createdAt)
 
+        callback?.(newTranslationDocument, newMetadataDocument, client)
```

Comment:
Maybe add a `catch((err) => {})` here. Undecided whether this should toast with an error or log to console, but it would be good to handle this in some way.

Reply:
Yes you might be right actually. Might be confusing to let the error get to the `catch` case for creating the actual translation. 

You think an empty catch would suffice for now?

Reply:
maybe a `console.error()` would be better than an empty catch

Reply:
Sorry I was a bit slow to edit my comment. Opted for using a toast to make it very clear that something went wrong. Should make it very clear, even to non-technical users, that something is amiss.